### ISSUE TITLE
Can this work with less than 60 GB RAM and VRAM?

### ISSUE THREAD

**Issue by GithubStable** [Fanghua-Yu/SUPIR#37]
If yes can I find instructions?

**Comment by FurkanGozukara** [comment]
yes even works with 12 GB without LLaVA they recently added new options

parser.add_argument("--loading_half_params", action='store_true', default=False)
parser.add_argument("--use_tile_vae", action='store_true', default=False)

I also published [1 click installer app](https://www.patreon.com/posts/supir-1-click-99176057?cid=130855314) that supports 12 GB gpus with extra options

![image](https://github.com/Fanghua-Yu/SUPIR/assets/19240467/4cc4fb28-ba94-415a-b4ea-6ec0122b8541)

**Comment by moebiussurfing** [comment]
awesome @FurkanGozukara !
that's only for the 5$ Patreon tier, right?

**Comment by aartikov** [comment]
What does LLaVA do and why we could disable it?

**Comment by GithubStable** [comment]
Interesting question! We need more explanations indeed

**Comment by FurkanGozukara** [comment]
> awesome @FurkanGozukara ! that's only for the 5$ Patreon tier, right?

yes 5$ tier

@aartikov I already disabled it. LLaVA for image captioning but I have better LLaVA script and even much lesser VRAM scripts such as Kosmos 2. All shared in this post : https://www.patreon.com/posts/sota-image-for-2-90744385
captioners_clip_interrogator_v2.zip
LLaVA_auto_install_v3.zip
Qwen-VL_v3.zip
blip2_captioning_v1.zip
CogVLM_v7.zip
Kosmos-2_v5.zip

you don't even need an image captioning. you can just write captions yourself

@GithubStable any other questions you have please ask ty

**Comment by moebiussurfing** [comment]
Thanks @FurkanGozukara , 
PS
do you have an email or preferred inbox (twitter/ig) to write you some questions about subscribing to the Patreon?
I want to know if you have good stuff for two or three topics I am thinking of learning.
Regards

**Comment by FurkanGozukara** [comment]
> Thanks @FurkanGozukara , PS do you have an email or preferred inbox (twitter/ig) to write you some questions about subscribing to the Patreon? I want to know if you have good stuff for two or three topics I am thinking of learning. Regards

well i have email : monstermmorpg@gmail.com

or discord which i prefer : https://discord.com/servers/software-engineering-courses-secourses-772774097734074388

we have more than 6500 discord members
